setup.cfg

```diff
@@ -1,5 +1,9 @@
 [pytest]
-addopts = -v --cov zdict --cov-report html --cov-report term
+flake8-ignore =
+    zdict/__init__.py ALL
```

Comment:
We don't have any content in other `__init__.py` files currently.
Besides, I don't think we will add things into new `__init__.py` files so often.
So, maybe just ignore `zdict/__init__.py` at this time?